components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -677,6 +677,9 @@
 .error_updating_consents {
   margin-top: 1em;
   margin-left: 2em;
+}
+.view-title-required-not-granted {
+  font-size: 1.15em;
```

Comment:
1.15em is a magic value and is kind of a coincidence that it matches the other warning font size (which is not specified in em). A better fix is to make sure both fonts are explicitly set to the same value, whatever it is